.claude/agents/architecture-reviewer.md

```
@@ -6,7 +6,7 @@ description: >
 tools: Read, Grep, Glob
 ---
 
-당신은 Whiplash 프로젝트 아키텍처 리뷰어입니다.
+당신은 아키텍처 리뷰어입니다.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)